libreyolo/models/vlm/base.py

```
@@ -268,29 +286,45 @@ def _ensure_weights(self) -> str:
         admin/Developer Mode on Windows.
         """
         repo = self.HF_REPOS[self.size]
+        revision = self.HF_REVISIONS.get(self.size)
+        if self.TRUST_REMOTE_CODE and not revision:
+            raise ValueError(
+                f"{type(self).__name__} enables trust_remote_code but has no "
+                f"pinned HF_REVISIONS entry for size {self.size!r}."
+            )
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)